### PR TITLE
Bugfix Returning Typenames

### PR DESCRIPTION
**Pull request by OberstVonGatow**
The returning Typenames concat wit 'ret_' and abapname. If abapname is 30 characters, ret_abapname will be 34 and we get a Syntaxerror in the Interface

### CODE REVIEW

File: src/v2/zcl_oapi_generator_v2.clas.abap [hunk 1]
```diff
@@ -345,14 +345,14 @@ CLASS zcl_oapi_generator_v2 IMPLEMENTATION.
   METHOD find_returning_parameter.
     DATA ls_response LIKE LINE OF is_operation-responses.
     DATA ls_content LIKE LINE OF ls_response-content.
-    DATA lv_typename TYPE string.
+    DATA lv_typename TYPE seoclsname.
```
Comment (larshp) [inline src/v2/zcl_oapi_generator_v2.clas.abap:348]: `seoclsname` is not released, https://abapedia.org/steampunk-2305-api/

**Comment by larshp** [comment]
please add a unit test, and/or another test case to https://github.com/abap-openapi/abap-openapi/tree/main/test_v2

**Comment by OberstVonGatow** [comment]
> please add a unit test, and/or another test case to https://github.com/abap-openapi/abap-openapi/tree/main/test_v2

Unit Test added.
DATA(...) Should be upper case im my opinion.

**Comment by larshp** [comment]
> > please add a unit test, and/or another test case to https://github.com/abap-openapi/abap-openapi/tree/main/test_v2
> 
> Unit Test added. DATA(...) Should be upper case im my opinion.

the code is https://github.com/abap-openapi/abap-openapi/blob/main/abaplint.json#L15

so "data(len)" is the only valid syntax in 702 where data is the variable name, "DATA(definition)" is from 740 something

**Review by larshp — APPROVED** [review]

**Comment by larshp** [comment]
thanks 👍

**Pull request merged**